### DISCUSSION TITLE
How to diable menu item?

### DISCUSSION THREAD

**Discussion by rongzhou** [ArthurSonzogni/FTXUI#507] [Q&A]
Any one know how to disable single menu item in proper way?

**Comment by ArthurSonzogni** [comment]
There are many ways. What do you means by "disable". You mean adding a new boolean state "enabled" and depending on this it would:
- not display the entry at all
- display the entry with a gray color and make it non selectable.

**Reply by in-text** [reply]
Hello! This answer interests me too. How can I display the entry, let's say a checkbox, and make it not selectable? For example: I want to display internet connection status via checkbox.
